jsdom/jsdom.ts

```diff
@@ -0,0 +1,15 @@
+import "./vendor/jsdom.js";
+
+// TODO: Investigate how to reuse typings from:
+// - https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jsdom/index.d.ts
+// - https://github.com/microsoft/TypeScript/blob/master/lib/lib.dom.d.ts
```

Comment:
@kitsonk Any idea on how to reuse these types here?

Reply:
There isn't really a way at the moment.  It is covered by denoland/deno#1432.

Reply:
Ah, too bad. I guess I will add some preliminary types for now, and we can improve them later, once `.d.ts` files are supported.